test/reporters/base.spec.js

```diff
@@ -403,4 +404,10 @@ describe('Base reporter', function() {
     errOut = stdout.join('\n').trim();
     expect(errOut, 'to be', '1) test title:\n     Error\n  foo\n  bar');
   });
+
+  it('should let you stub out console.log without effecting reporters output', function() {
+    sinon.stub(console, 'log');
+    Base.list([]);
+    expect(console.log, 'was not called');
+  });
```

Comment:
Shouldn't this be done twice, once normally expecting "to call", then apply your hideaway and expect "not to call"?

Reply:
Dont quite follow the "to call" scenario. Stub console.log before importing `Base` to prove that it is called?

Reply:
@plroebuck sorry still don't quite follow what you meant above, could you explain please.

Reply:
This test uses unproven assumption that `Base.list` implementation uses `console.log` to begin with. In order to prove your test stubs it out, you have to prove the opposite as well.

```js
 it('should let you stub out console.log without effecting reporters output', () => {
    sinon.stub(console, 'log');
    sinon.stub(Base, 'consoleLog');
    Base.list([]);
    expect(Base.consoleLog, 'was called');
    expect(console.log, 'was not called');
  });
```

Reply:
Yep..done.